tripal_ws/includes/TripalWebService/TripalContentService_v0_1.inc

```diff
@@ -90,15 +90,19 @@ class TripalContentService_v0_1 extends TripalWebService {
       $this->doEntityList($ctype);
     }
     // If we have an entity ID then build the resource for a single entity.
-    else if ($ctype and $entity_id and !$expfield) {
-      $this->doEntity($ctype, $entity_id);
-    }
-    else if ($ctype and $entity_id and $expfield) {
-      $this->doExpandedField($ctype, $entity_id, $expfield);
-    }
-    // Otherwise just list all of the available content types.
     else {
-      $this->doContentTypesList();
+      if ($ctype and $entity_id and !$expfield) {
```

Comment:
Just curious why this if statement was changed?

Reply:
its an automated style change.

the original:

```
else if
else if
else
```

the change:

````
else 
if 
else 
if 
else
```

If youre concerned about the style changes I can remake the PR with just the inserted line.  Its hard to remember NOT to run my formatter when changing files...

Reply:
As you can see its a recommendation of the drupal coder code sniffer

![screen shot 2018-12-17 at 11 45 48 am](https://user-images.githubusercontent.com/7063154/50101725-59644380-01f1-11e9-8ea3-a4867ccfd7bf.png)

Reply:
If I'm reading this right, it looks like the else if's from the two lines above got merged into a single else.... I'm not sure that's the same logic....

Reply:
look at lines 95-105

```php
    else {
      if ($ctype and $entity_id and !$expfield) {
     $this->doEntity($ctype, $entity_id);
      }
      else {
        if ($ctype and $entity_id and $expfield) {
          $this->doExpandedField($ctype, $entity_id, $expfield);
        }
        // Otherwise just list all of the available content types.
        else {
          $this->doContentTypesList();
        }
      }
```

youre right the elsifs got merged but they are branched in that else within if statements.  It looks like the same logic to me.

Reply:
Just to satisfy my paranoia can we put it back the way it was?  I don't have time to test that this reworking of those if statements is the same.  I can test your permissions fix easy enough.

Reply:
no problem.  but if its alright with you im going to revert the whole file: that way ill be more confident im not changing hte logic :)

Reply:
Yep sounds good.